packages/mux-player/src/index.ts

```diff
@@ -678,12 +678,19 @@ class MuxPlayerElement extends VideoApiElement {
     }
   }
 
+  /**
+   * Return the storyboard URL when a playback ID is provided,
+   * we aren't an audio player and the stream-type isn't live.
+   */
   get storyboard() {
     if (
       this.playbackId &&
       !this.audio &&
-      this.streamType &&
-      ![StreamTypes.LIVE, StreamTypes.LL_LIVE, StreamTypes.DVR, StreamTypes.LL_DVR].includes(this.streamType as any)
+      (!this.streamType ||
+        (this.streamType &&
```

Comment:
nit (non-blocking): `!this.streamType ||` the right hand of this condition already means `this.streamType` is not null so no need for `(this.streamType && )`